pieforproviders/pieforproviders
# [PIE-2049] inactive bug

## 💅🏼 What issue does this fix?


Closes #2049 - the name of the inactive reason had a typo so the rails validation was failing it

## 🍂 Before You Submit


## 🧳 Steps to test


1. Go to the dashboard
2. Click on 'Mark inactive'
3. Select 'Child is no longer receiving subsidies' as the reason
4. Select any date & click Save
5. The child should be inactive
